com.giuspen.cherrytree.json

```diff
@@ -21,6 +21,9 @@
     "/share/doc/",
     "/man/"
   ],
+  "skip-arches": [
```

Comment:
the skip-arches should go in a flathub.json file, to limit which builders to use and not in the manifest itself